src/components/category/Category.jsx

```diff
@@ -1,57 +1,45 @@
-import React from "react";
+import { useState, useEffect } from "react";
+import axios from "axios";
 import { Link } from "react-router-dom";
 
 const Category = () => {
+  const [categoryList, setCategoryList] = useState([]);
+
+  const getCategories = async () => {
+    try {
+      const { data } = await axios.get("/api/categories");
+      console.log(data.categories);
+      setCategoryList(data.categories);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  useEffect(() => {
+    getCategories();
+  }, []);
```

Comment:
good to see that you have made a function for getting category and called inside use effect. I liked this code breaking

Reply:
thanks